src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -55,6 +63,67 @@ private String getDeserializedConfig(MessageOrBuilder message) {
     return Base64.getEncoder().encodeToString(channelConfigString.getBytes());
   }
 
+  @Override
+  public ResponseEntity<ClientResponseModel> getAnchorPeerForChannel(String channelName) {
+    Network network = gateway.getNetwork(channelName);
+    Set<String> anchorPeersSet = new HashSet<>();
```

Comment:
[nitpick] declare just before where this Set will get used.

Reply:
Setting it here as it  is getting used outside the try-catch blocks as well.

Reply:
Addressed, Thanks :)